src/temperature.h

```diff
@@ -0,0 +1,10 @@
+#ifndef TEMPERATURE_H
+#define TEMPERATURE_H
+
+#include "Arduino.h"
+#include "mcu_pinout.h"
+
+// Actuating the heater element
+void setBoiler(int val);
```

Comment:
Great care should be taken to fensure that this is always inlined and is static, to reduce code size and improve performance.